CHANGELOG.md

```diff
@@ -18,7 +18,9 @@
 1. [15749](https://github.com/influxdata/influxdb/pull/15749): Expose bundle analysis tools for frontend resources
 1. [15674](https://github.com/influxdata/influxdb/pull/15674): Allow users to view just the output section of a telegraf config
 1. [15923](https://github.com/influxdata/influxdb/pull/15923): Allow the users to see string data in the single stat graph type
-1. [16057](https://github.com/influxdata/influxdb/pull/16057): Adds `properties` to each cell on GET /dashboards/{dashboardID}
+1. [15314](https://github.com/influxdata/influxdb/issues/15314): Allow the users to see string data in the single stat graph type
+1. [16057] (https://github.com/influxdata/influxdb/pull/16057): Adds `properties` to each cell on GET /dashboards/{dashboardID}
+1. [16112] (https://github.com/influxdata/influxdb/pull/16112): Create cell view properties on dashboard creation
```

Comment:
1. the space between the link name and the link (`[16057] (https...`) will cause the link to not render, so you'll want to smash them together: 
    ```md
      [16057](https://github.com/influxdata/influxdb/pull/16057): Adds `properties` to each cell on GET /dashboards/{dashboardID}

      [16112](https://github.com/influxdata/influxdb/pull/16112): Create cell view properties on dashboard creation
    ```
1. these changes are in the wrong section - we've already released this version. they need to be moved [up above](https://github.com/influxdata/influxdb/pull/16112/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3) to the `v2.0.0-alpha.21 [unreleased]` release section

Reply:
ah, good call! these got borked on my end after I rebased. i'll update